docs/source/configuration/key_value_service_configs/oracle_key_value_service_config.rst

```diff
@@ -19,7 +19,7 @@ e.g.
 
 .. code-block:: groovy
 
-  runtime 'com.palantir.atlasdb:atlasdb-dbkvs:0.19.0'
+  runtime 'com.palantir.atlasdb:atlasdb-dbkvs:0.25.0'
```

Comment:
Oh yeah, definitely! We can store this inside the `|release|` replacement, which is in `conf.py`; but we'll need to set up some magic to get conf.py to auto-update. At the moment I see it's just hard-coded to develop.